duneggd/Config/ArgonCube/ArgonCubeDetector.cfg

```diff
@@ -1,114 +1,81 @@
-###############################
-# ArCLight
-###############################
 [ArCLight]
-class                   = duneggd.ArgonCube.ArCLight.ArCLightBuilder
-subbuilders             = []
-WLS_dimension           = {'dx':Q('464.215mm')/2,'dy':Q('299.599mm')/2,'dz':Q('10mm')/2,'tpb_dd':Q('0.1mm')/2,'dcm_dd':Q('0.11mm')/2}
-SiPM_dimension          = {'dx':Q('1.470mm')/2,'dy':Q('6.000mm')/2,'dz':Q('6.000mm')/2,'pitch':Q('47.000mm')/2}
-SiPM_Mask               = {'dx':Q('1.650mm')/2,'dy':Q('78.250mm')/2,'dz':Q('10mm')/2,'pitch':Q('299.599mm')/3/2}
-SiPM_PCB                = {'dx':Q('3.000mm')/2,'dy':Q('299.599mm')/3/2,'dz':Q('10mm')/2,'pitch':Q('299.599mm')/3/2}
-N_SiPM                  = 6
-N_Mask                  = 3
+class = duneggd.ArgonCube.ArCLight.ArCLightBuilder
+subbuilders = []
+WLS_dimension = {'dx':Q('457.977mm')/2,'dy':Q('299.599mm')/2,'dz':Q('10.0mm')/2,'tpb_dd':Q('0.1mm')/2,'dcm_dd':Q('0.11mm')/2}
+SiPM_dimension = {'dx':Q('1.470mm')/2,'dy':Q('6.0mm')/2,'dz':Q('6.0mm')/2,'pitch':Q('47.0mm')/2}
+SiPM_Mask = {'dx':Q('1.650mm')/2,'dy':Q('78.25mm')/2,'dz':Q('10.0mm')/2,'pitch':Q('299.599mm')/3/2}
+SiPM_PCB = {'dx':Q('3.000mm')/2,'dy':Q('299.599mm')/3/2,'dz':Q('10.0mm')/2,'pitch':Q('299.599mm')/3/2}
+N_SiPM = 6
+N_Mask = 3
 
-###############################
-# PixelPlane
-###############################
 [PixelPlane]
-class                   = duneggd.ArgonCube.PixelPlane.PixelPlaneBuilder
-subbuilders             = []
-PCB_dimension           = {'dx':Q('3.175mm')/2,'dy':Q('299.599mm')/2,'dz':Q('478.318mm')/2}
-Pixel_dimension         = {'dx':Q('0.051mm')/2,'dy':Q('4.000mm')/2,'dz':Q('4.000mm')/2}
-Asic_dimension          = {'dx':Q('1.422mm')/2,'dy':Q('16.000mm')/2,'dz':Q('16.000mm')/2}
-N_Pixel                 = 0
-#N_Pixel                 = 70
-N_Asic                  = 10
+class = duneggd.ArgonCube.PixelPlane.PixelPlaneBuilder
+subbuilders = []
+PCB_dimension = {'dx':Q('3.175mm')/2,'dy':Q('299.599mm')/2,'dz':Q('477.85999999999996mm')/2}
+Pixel_dimension = {'dx':Q('0.051mm')/2,'dy':Q('4.000mm')/2,'dz':Q('4.000mm')/2}
+Asic_dimension = {'dx':Q('1.422mm')/2,'dy':Q('16.0mm')/2,'dz':Q('16.0mm')/2}
+N_Pixel = 0
+N_Asic = 10
```

Comment:
A lot of changes are just you removing spaces. Could you keep with the established config formatting (i.e. aligned equal signs) and only change the lines where you actually change something? Always better to keep things consistent. And you won't get git-blamed for lines where you did not actually change any parameters. ;)

Removing commented out configuration lines is totally fine though. Keeping comments of how things used to be before a change is a bit useless when we have the complete commit history.